pubpub/pubpub
# Add support for PDF and TIFF images in imports

This PR adds importer support for .pdf and .tif(f) files, by way of converting them to .png files on import. I anticipate occasional confusion from folks trying to upload PDFs for their text content, so we'll show this warning when they are queued up without a document we can import:

<img width="471" alt="image" src="https://user-images.githubusercontent.com/2208769/83792830-a0eec180-a669-11ea-9028-0a36f80cd922.png">

I also enabled references to external image files with the file extension, which seems to be a pattern in LaTeX-world.

_Test plan:_
- Try uploading a file that references .pdf, .tiff, and currently supported (e.g. .png) images by name, with and without file extensions, and verify that they are imported correctly.
- Also verify that a .docx with embedded .tiff images works ([this one](http://assets.pubpub.org/_testing/21591292612062.docx) will do nicely)
- Check that the above error message appears when only .pdfs are queued for import.

Fixes #843.
